File: my-project/src/app/class/vertion.ts
import { Feature } from "app/class/feature";

export class Vertion {
	public name: string;
	public vertionID: number;
	public creationTime: Date;
	public featureList: Array<Feature>;

	constructor() {
	}

	toString() {
		return `${this.name} since ${this.creationTime} has ${this.featureList} features`;
    }
}

File: server/angular2mercas/Models/Vertion.cs
using angular2mercas.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Net;
using System.Net.Http;
using System.Web.Http;

namespace angular2mercas.Models
{
    public class Vertion
    {
        public int VertionID { get; set; }
        public string Name { get; set; }
		public DateTime CreationTime { get; set; }
		public List<Feature> FeatureList { get; set; }
		
    }
}